master/getting-started/kubernetes/installation/integration.md

```diff
@@ -33,7 +33,7 @@ There are three components of a Calico / Kubernetes integration.
 - The Calico per-node docker container, [calico/node](https://quay.io/repository/calico/node?tab=tags)
 - The [cni-plugin](https://github.com/projectcalico/cni-plugin) network plugin binaries.
   - This is the combination of two binary executables and a configuration file.
-- When using Kubernetes NetworkPolicy, the Calico policy controller is also required.
+- When using Kubernetes NetworkPolicy, you must also deploy the Calico kubernetes controllers.
```

Comment:
"When using Kubernetes NetworkPolicy, you must also deploy the Calico kubernetes controllers." --> "When using Kubernetes network policy, you must also deploy the Calico Kubernetes controllers."